packages/harpguru-core/src/utils/reduce-layoutfacts-to-dynamicsizes/reduce-layoutfacts-to-dynamicsizes.ts

```diff
@@ -38,15 +27,17 @@ const relativeSizes: Omit<
   10: 76.013,
   11: 122.989,
 } as const
-
 const relativeColumnWidth = 9
 const relativeFragmentGutterWidth = 7
 const relativeLabelProtrusion = 9
 const relativeLabelIconSize = 7
 
-export const useSizes = (): SizeSchemes => {
+export const reduceLayoutFactsToDynamicSizes = (
```

Comment:
A test file might be feasible given the small scope of input parameters.

On the other hand, one might not be crucial given the sizes critical role within the snapshots. I would only consider this alternative given the tight timeframe and would raise techdebt to fill in.

Reply:
Having said all this, I don't think a unit test is feasible. There is nothing about the tests which will make any sense to a human. The output object is too complex and volatile. It will change with almost all possible inputs but won't do so in a way which makes any sense to the reader. It's a lot of work for little benefit.